src/PIL/Image.py

```diff
@@ -2871,7 +2884,14 @@ class ImageTransformHandler:
     (for use with :py:meth:`~PIL.Image.Image.transform`)
     """
 
-    pass
+    @abc.abstractmethod
+    def transform(
+        self,
+        size: tuple[int, int],
+        image: Image,
+        **options: dict[str, str | int | tuple[int, ...] | list[int]],
+    ) -> Image:
+        pass  # pragma: no cover
```

Comment:
Let's also add `@abc.abstractmethod` to `exclude_also`:

```suggestion
        pass
```

And we can remove the pragma from `ImageShow`'s `@abc.abstractmethod` too.